### DISCUSSION TITLE
Welcome to Reward-Your-Teacher-API Discussions!

### DISCUSSION THREAD

**Discussion by Valleon6** [Valleon6/Reward-Your-Teacher-API#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Valleon96** [comment]
Hello me on the other side.
Currently working on the boiler plates for the project.
